sklearn/externals/funcsigs.py

```diff
@@ -20,8 +20,8 @@
 __all__ = ['BoundArguments', 'Parameter', 'Signature', 'signature']
 
 
-_WrapperDescriptor = type(type.__call__)
```

Comment:
Np, but which one is the final upstream? The link you gave me: https://github.com/testing-cabal/funcsigs/, or here: https://github.com/aliles/funcsigs ? Thanks! @ogrisel